src/gobapi/dump/csv.py

```diff
@@ -83,8 +83,20 @@ def _csv_values(value, spec):
     if spec['type'] in REFERENCE_TYPES:
         return _csv_reference_values(value, spec)
     elif spec['type'] == 'GOB.JSON':
-        value = value or {}
-        return [_csv_value(value.get(field)) for field in spec['attributes'].keys()]
+        if type(value) == list:
+            values = []
+            value = value or []
```

Comment:
Isn't this always the case? Value is of type list, so it's either a list with length > 0 or []